### DISCUSSION TITLE
What purpose does the Large Mining Drill fluid storages serve?

### DISCUSSION THREAD

**Discussion by TheChosenIndividual** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1678] [Q&A] [answered]
Simple, i realized that the Large Mining Drill still has fluid storages for acids, so, i wanna know, what are they for? speeding up bedrock ore extraction or something else?

**Comment by Hacker6400** [comment, marked as answer]
It's for the old bedrock ore that are still in the game

**Reply by TheChosenIndividual** [reply]
I mean...yeah, i know that but...any use in the modern system? or...
